src/EnergyPlus/SimulationManager.cc

```diff
@@ -1013,6 +1013,27 @@ namespace SimulationManager {
             }
         }
 
+        state.dataHeatBalMgr->CurrentModuleObject = "OutputControl:ResilienceSummaries";
```

Comment:
In the future, there's no need to use a state variable to hold this string.  Just keep a constexpr string view variable of it and use that.

Reply:
I'll remember and use string view in the future.